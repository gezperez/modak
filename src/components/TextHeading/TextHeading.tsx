import React from 'react'
import { View } from 'react-native'

const TextHeading = () => {
  return <View />
}

export default TextHeading
