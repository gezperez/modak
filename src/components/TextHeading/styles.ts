import { StyleSheet } from 'react-native'

const styles = StyleSheet.create({
  uppercase: {
    textTransform: 'uppercase',
  },
  alignCenter: {
    textAlign: 'center',
  },
  oneLineFlex: {
    flexShrink: 1,
  },
})

export default styles
