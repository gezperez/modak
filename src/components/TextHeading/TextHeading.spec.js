import React from 'react'
import { render } from '@testing-library/react-native'
import TextHeading from './TextHeading'

describe('TextHeading', () => {
  it('shold match snapshot', () => {
    const { toJSON } = render(<TextHeading />)
    expect(toJSON()).toMatchSnapshot('TextHeading')
  })
})
