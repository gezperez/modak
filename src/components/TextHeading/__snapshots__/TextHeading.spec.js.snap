// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextHeading shold match snapshot: TextHeading 1`] = `<View />`;
